clients/venice-admin-tool/src/main/java/com/linkedin/venice/datarecovery/StoreRepushCommand.java

```diff
@@ -158,6 +201,16 @@ public void setExtraCommandArgs(String args) {
     public void setSourceFabric(String fabric) {
       this.sourceFabric = fabric;
     }
+
+    public void setTimestamp(String timestamp) {
+      DateTimeFormatter formatter = DateTimeFormatter.ofPattern("MM-dd-yyyy HH:mm:ss");
```

Comment:
Let's also use the [DateTimeFormatter.ISO_LOCAL_DATE_TIME](https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html#ISO_LOCAL_DATE_TIME) constant here.